api/views.py

```diff
@@ -65,7 +67,44 @@ class RegistrationAPIViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
     serializer_class = RegistrationSerializer
 
 
+class VolunteerFilter(filters.FilterSet):
+    OPERATORS = {"eq": eq, "gt": gt, "lt": lt}
+
+    times_volunteered = filters.NumberFilter(method='get_times_volunteered', field_name='eq')
+    times_volunteered__gt = filters.NumberFilter(method='get_times_volunteered', field_name='gt')
+    times_volunteered__lt = filters.NumberFilter(method='get_times_volunteered', field_name='lt')
+
+    def get_times_volunteered(self, queryset, field_name, value, ):
```

Comment:
Isn't it easier to simply perform an aggregate instead of iterating all volunteers?
Also performance wise.

Reply:
Didn't understand you completely. @stavlocker What do you think?
More about the solution: that's a solution I found in Stackoverflow for "patching" it, because this field is a python method and can't be accessed with "traditional" filtering

Reply:
The python method calls a synchronous query for each voluteer, which would cause a lot of latency issues.
Consider something in the likes of this, as part of building the queryset:
>>> pubs = Publisher.objects.annotate(num_books=Count('book'))

More info:
https://docs.djangoproject.com/en/3.0/topics/db/aggregation/